pay-admin/pyproject.toml

```diff
@@ -25,7 +25,7 @@ jinja2 = "^3.1.3"
 pay-api = {git = "https://github.com/bcgov/sbc-pay.git", subdirectory = "pay-api"}
 sbc-common-components = {git = "https://github.com/bcgov/sbc-common-components.git", subdirectory = "python"}
 simple-cloudevent = {git = "https://github.com/daxiom/simple-cloudevent.py.git"}
-flask-jwt-oidc = {git = "https://github.com/thorwolpert/flask-jwt-oidc.git"}
```

Comment:
I'll clean this up today, don't need any of these just `pay-api` it already includes all of these